README.md

```diff
@@ -1,37 +1,101 @@
 # pg-backup-api
-A server that provides an HTTP API to interact with Postgres backups
+
+A server that provides an REST API to interact with Barman.
 
 ## Installation
-tbd
+
+This tool is supposed to be installed in the same server where Barman is already
+set up and running.
+
+### Package installation
+
+Configure EDB repos 2.0 and install the package `pg-backup-api` using your
+package manager.
+
+### Installation from source code
+
+1. Clone this repository, e.g.:
+
+```bash
+git clone https://github.com/EnterpriseDB/pg-backup-api.git
+```
+
+2. Install `pg-backup-api` and its requirements:
+
+```bash
+cd pg-backup-api/pg_backup_api
+sudo python3 setup.py install
+```
+
+**Note:** `pg_config` application should be in your `PATH` because `psycopg2` is
+in the dependency tree. Besides that, you also need the `libpq` C library in
+your system (`libpq-dev` package on Debian based, or `libpq-devel` package on
+RedHat based).
 
 ## Running the Flask app
 
 ### Manually
 
-Install `pg-backup-api` and change to the Barman user, then run
+As Barman user run:
 
-`pg-backup-api serve` 
+```bash
+pg-backup-api serve
+```
 
-or `pg-backup-api serve --port N` if you want to use a different port than the default (which is 7480).
+**Note:** by default `pg-backup-api` runs on port `7480`. You can override that
+behavior by passing `--port N` to `serve` command, `N` being the port to listen
+on.
+
+The above comand will start up the REST API as a development server. If you
+want to run the REST API as an WSGI application, use the approach described in
+the `Service` section.
 
 ### Service
 
-If running from a local source directory, copy `pg-backup-api/packaging/pg-backup-api.service` to `/etc/systemd/system/`. If you've installed
-pg-backup-api as a package, installation will put the service file in the right place for you.
+If you intend to run `pg-backup-api` from a local source directory, then you
+need to set up the `pg-backup-api` service. You need to copy a couple files
+from the [pg-backup-api-packaging repo](https://github.com/EnterpriseDB/pg-backup-api-packaging):
```

Comment:
This repo is private, so we shouldn't mention it in the public-facing docs

Reply:
The original comment on the docs is also incorrect for public consumption so we need something that would work for everyone